integration-tests/tests/snapd_test.go

```diff
@@ -64,6 +65,19 @@ func (s *snapdTestSuite) SetUpTest(c *check.C) {
 
 	s.cmd.Start()
 
+	done := false
+	for i := 0; i < 50; i++ {
+		if err := exec.Command("sudo", "chmod", "/run/snapd.socket", "0666").Run(); err == nil {
```

Comment:
As a side note this comments only makes sense if we don't use integration-tests/testutils/wait package as suggested by @elopio, it uses internally the cli package.